### PR TITLE
Get tests passing

### PR DESCRIPTION
**Pull request by scottito**
@nmaterise I had to comment Mattis-Bardeen from requirements.txt because it cannot be pip installed by symbol.
I disabled warnings for pylint which may change in the future but currently flags almost every line with strict pep-8 practices.
Numpy isclose works to compare expected values of fit.

I also moved minimum working example to root directory and included instructions on how to use in readme.

### CODE REVIEW

**Comment by nmaterise** [comment]
> @nmaterise I had to comment Mattis-Bardeen from requirements.txt because it cannot be pip installed by symbol. I disabled warnings for pylint which may change in the future but currently flags almost every line with strict pep-8 practices. Numpy isclose works to compare expected values of fit.
> 
> I also moved minimum working example to root directory and included instructions on how to use in readme.

That all sounds good. I think we will want to pull `Mattis-Bardeen` from its github. I think my code used `pip` from the cloned version of the code, and that's why it ended up in the requirements.txt file. If you know how to add github repositories to requirements.txt, I think it will fix that problem. Everything else looks good.

File: test/test_fit.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+def test_values():
+    # Perform a fit on the data with given parameters
+    my_resonator.fit_method(fit_type, MC_iteration, MC_rounds=MC_rounds, MC_fix=MC_fix, 
+                            manual_init=manual_init, MC_step_const=0.3)
+    output_params, conf_array, error, init = my_resonator.fit(None)
+    if np.isclose(output_params, [306209.29520741466, 928387.8353093242, 4512506964.96403, -0.15431421139237836]).sum() != 4:
+        raise Exception("Different output parameters than expected in user_test.py")
+    if np.isclose(conf_array, [588.1821370752295, 1685.6238136050524, 1634.941474343068, 1654.6028376782779, 0.0017515102820822026, 14.244794845581055]).sum() != 6:
+        raise Exception("Different confidence array than expected in user_test.py")
+    if np.isclose(error, [0.00012610664079341314]).sum() != 1:
```
Comment (nmaterise) [inline test/test_fit.py:42]: I think you can replace `np.isclose().sum()` with `np.allclose()`.

Comment (scottito) [reply test/test_fit.py:42]: Good idea

Comment (scottito) [reply test/test_fit.py]: Done.

**Review by nmaterise — APPROVED** [review]

**Pull request merged**